### PR TITLE
Remove redundant "read:targets" scope

### PR DESCRIPTION
**Pull request by MVrachev**
Remove redundant "read:targets" scope.

Closes https://github.com/vmware/repository-service-tuf-api/issues/177

Signed-off-by: Martin Vrachev <mvrachev@vmware.com>

### CODE REVIEW

File: tests/unit/api/test_tasks.py [hunk 1]
```diff
@@ -75,7 +75,7 @@ def test_get_invalid_scope(self, test_client, monkeypatch):
         token_payload = {
             "username": "admin",
             "password": "secret",
-            "scope": ("read:targets " "read:settings " "read:token "),
+            "scope": (),
```
Comment (kairoaraujo) [inline tests/unit/api/test_tasks.py:78]:
```suggestion
            "scope": "",
```

**Review by kairoaraujo — CHANGES_REQUESTED** [review]

File: docs/source/guide/api/swagger.json [hunk 2]
```diff
@@ -505,7 +505,7 @@
                     "scope": {
                         "title": "Scope",
                         "type": "string",
-                        "description": "Add scopes separeted by space. Default user scopes. Users might not have all scopes. Available scopes: read:bootstrap read:settings read:targets read:tasks read:token write:bootstrap write:targets write:token delete:targets",
+                        "description": "Add scopes separeted by space. Default user scopes. Users might not have all scopes. Available scopes: read:bootstrap read:settings read:tasks read:token write:bootstrap write:targets write:token delete:targets",
```
Comment (kairoaraujo) [inline docs/source/guide/api/swagger.json:508]: You don't need to change swagger.json.
It is done automatically when changed the code running `make docs`

It comes from here:
https://github.com/vmware/repository-service-tuf-api/blob/a41294fba0dd2a6c412f271f582b23e106570c8c/repository_service_tuf_api/token.py#L28-L33

Comment (MVrachev) [reply docs/source/guide/api/swagger.json:508]: I used `make docs` locally to generate the changes again.

File: tests/unit/api/test_tasks.py [hunk 3]
```diff
@@ -75,7 +75,7 @@ def test_get_invalid_scope(self, test_client, monkeypatch):
         token_payload = {
             "username": "admin",
             "password": "secret",
-            "scope": ("read:targets " "read:settings " "read:token "),
+            "scope": ("read:settings"),
```
Comment (kairoaraujo) [inline tests/unit/api/test_tasks.py:78]:
```suggestion
            "scope": "read:settings",
```
The scope is a string

File: tests/unit/api/test_token.py [hunk 4]
```diff
@@ -145,7 +145,7 @@ class User:
                 id=1213,
                 username="read_user",
                 password="test",
-                scopes=[Scope("read:targets")],
+                scopes=[],
```
Comment (kairoaraujo) [inline tests/unit/api/test_token.py:148]: nit: It mocks the database with a "read_user" -- even our authentication/authorization built-in feature doesn't support multiple users (only "admin")
Do we want to test it with a user that doesn't have any scope?
On line 164 we request a token for a user with `write:targets`. I would suggest giving the user at least one valid scope (`read:settings`) for example.

**Comment by MVrachev** [comment]
@kairoaraujo it should be okay now.

File: tests/unit/api/test_token.py [hunk 5]
```diff
@@ -145,7 +145,7 @@ class User:
                 id=1213,
                 username="read_user",
                 password="test",
-                scopes=[Scope("read:targets")],
+                scopes="read:settings",
```
Comment (kairoaraujo) [inline tests/unit/api/test_token.py:148]: To fix the CI error, You need to revert it. It is the Database object Mock.

Comment (MVrachev) [reply tests/unit/api/test_token.py:148]: Ohh, sorry, obvious mistake...

**Review by kairoaraujo — APPROVED** [review]

**Review by rdimitrov — DISMISSED** [review]

**Comment by kairoaraujo** [comment]
@MVrachev, a kindly reminder we need to rebase with the main. 😄

**Comment by MVrachev** [comment]
I know, I had local problems with `plantuml` because of which I can't generate a new `swagger.json`.

**Comment by MVrachev** [comment]
@kairoaraujo it's ready for another approval.

**Review by kairoaraujo — APPROVED** [review]

**Comment by kairoaraujo** [comment]
Merging as it was just waiting for the rebase.

**Pull request merged**